src/prommis/nanofiltration/diafiltration_two_salt.py

```diff
@@ -364,212 +407,226 @@ def add_variables(self):
 
         # add global variables
         self.membrane_width = Var(
-            initialize=1,
+            initialize=4,
             units=units.m,
-            bounds=[0, None],
+            bounds=[1e-11, None],
             doc="Width of the membrane (x-direction)",
         )
         self.membrane_length = Var(
-            initialize=100,
+            initialize=40,
             units=units.m,
-            bounds=[0, None],
+            bounds=[1e-11, None],
             doc="Length of the membrane, wound radially",
         )
         self.applied_pressure = Var(
-            initialize=10,
+            initialize=15,
             units=units.bar,
-            bounds=[0, None],
+            bounds=[1e-11, None],
```

Comment:
@agarciadiego , do we have the IPOPT option `bound_push=0` for this repo's default settings?